includes/settings/tools.php

```diff
@@ -307,8 +307,9 @@ protected function create_tabs() {
 							$intro_text = '<p>' . $intro_text . '</p>';
 							$newsletter_opt_in_text = sprintf( esc_html__( '<a id="beta-tester-first-to-know" href="%s">Click here</a> to join our first-to-know email updates.', 'elementor' ), '#' );
 
-							echo $intro_text;
-							echo $newsletter_opt_in_text;
+							// PHP Lint: Ignore statements added because these variables are escaped above.
+							echo $intro_text; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+							echo $newsletter_opt_in_text; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Add a // PHPCS comment above the second echo.

Reply:
Why? The comment above the first one refers to both echo statements.